tiny_skia/src/backend.rs

```diff
@@ -31,57 +34,127 @@ impl Backend {
 
             #[cfg(feature = "svg")]
             vector_pipeline: crate::vector::Pipeline::new(),
+
+            last_primitives: Vec::new(),
```

Comment:
If a single Backend were able to be used multiple times (for multiple windows if that was intended) you would end up with a situation where the second window is fully redrawn because the primitives may be completely different on that other window.